subgraph/config/kovan.json

```diff
@@ -0,0 +1,5 @@
+{
+  "network": "kovan",
+  "ExchangeV3Address": "0x4eacd0af335451709e1e7b570b8ea68edec8bc97",
```

Comment:
this is the official 0x address for kovan right?

Reply:
yep